### ISSUE TITLE
Inverted sky block on ground looks all weird

### ISSUE THREAD

**Issue by zeldamon** [OpenMods/OpenBlocks#763]
When a skyblock is placed, if it is anywhere within sight, the whole screen turns into the sky. With a shaders on, it just looks strange.

![2016-03-23_14 04 01](https://cloud.githubusercontent.com/assets/11427157/13995650/63ead684-f100-11e5-95f4-e0307045a138.png)

![2016-03-23_14 04 16](https://cloud.githubusercontent.com/assets/11427157/13995626/52e4090a-f100-11e5-9534-ce5882523877.png)

**Comment by boq** [comment]
It is bug, but probably will never be fixed. We are not able to guarantee compatibility with mods that deeply change rendering engine, especially closed-source.

**Comment by zeldamon** [comment]
> It is bug, but probably will never be fixed. We are not able to guarantee compatibility with mods that deeply change rendering engine, especially closed-source.

However, even without shaders, the game is unplayable with it as nothing is visible.

**Comment by boq** [comment]
But I can see you have this mod installed on both screenshots. Even when it's disabled, it still has to make some changes to pipeline to make it possible to add shaders.

**Comment by zeldamon** [comment]
> But I can see you have this mod installed on both screenshots. Even when it's disabled, it still has to make some changes to pipeline to make it possible to add shaders.

You are correct. After commenting, but before you replied, I found that removing the shader's core fixed it. By any chance, are you aware of a shader's core that doesn't have this issue?
